perl-Bootloader-testsuite/tests/test_interface/backup/fake_root1/boot/extlinux/extlinux.conf

```diff
@@ -0,0 +1,25 @@
+TIMEOUT 1000
+ONTIMEOUT localboot
+DEFAULT default
+MENU TITLE Boot menu
+PROMPT 0
+
+LABEL default
+         MENU LABEL Default
+         LINUX /boot/zImage
+         FDTDIR /boot/dtb/
+         FDT exynos5250-snow.dtb
```

Comment:
Maybe use fdt and fdtdir in independent sections? Or even issue a warning when parsing such a file, as it is not really well-formed?